### PR TITLE
unmarshal loaded addresses

### PR DESCRIPTION
**Pull request by ricebin**
which is needed for v0 transactions

https://docs.solana.com/developing/clients/jsonrpc-api#results-43

### CODE REVIEW

File: rpc/types.go [hunk 1]
```diff
@@ -150,6 +150,11 @@ type UiTokenAmount struct {
 	UiAmountString string `json:"uiAmountString"`
 }
 
+type LoadedAddresses struct {
+	ReadOnly []solana.PublicKey `json:"readOnly"`
+	Writable []solana.PublicKey `json:"writable"`
```
Comment (gagliardetto) [inline rpc/types.go:155]:
```suggestion
	ReadOnly solana.PublicKeySlice `json:"readOnly"`
	Writable solana.PublicKeySlice `json:"writable"`
```

File: rpc/types.go [hunk 2]
```diff
@@ -150,6 +150,11 @@ type UiTokenAmount struct {
 	UiAmountString string `json:"uiAmountString"`
 }
 
+type LoadedAddresses struct {
+	ReadOnly solana.PublicKeySlice `json:"readOnly"`
```
[the remark this answers is not in the document]
Comment (ricebin) [reply rpc/types.go]: interesting.. thanks for noticing
it seems golang handles this .. cuz it seems to deserialize for me

thanks!

[the remark this answers is not in the document]
Comment (gagliardetto) [reply rpc/types.go]: Yep, go just doesn't care about the casing :laughing:

**Comment by gagliardetto** [comment]
Thanks

**Pull request merged**